### PR TITLE
chore: Package updates for vulnerabilities

### PR DESCRIPTION
**Pull request by dckesler**

### CODE REVIEW

File: packages/use-contractkit/src/contract-kit-reducer.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { Connector, Dapp, Network } from './types';
-import { localStorageKeys } from './constants';
 import { CONNECTOR_TYPES, UnauthenticatedConnector } from './connectors';
```
Comment (dckesler) [inline packages/use-contractkit/src/contract-kit-reducer.ts:1]: Linter changes

File: packages/use-contractkit/src/use-contract-kit-methods.ts [hunk 2]
```diff
@@ -33,7 +33,9 @@ export function useContractKitMethods(): ContractKitMethods {
           nextConnector.type,
           e
         );
-        dispatch('setConnectorInitError', e);
+        const error =
```
Comment (dckesler) [inline packages/use-contractkit/src/use-contract-kit-methods.ts:36]: For some reason typescript recognized this `catch` parameter to be the type `unknown` which makes sense. Don't know why it didn't catch this before.

**Review by jmrossy — APPROVED** [review]
Does this also solve the tar high sev security alert?

**Comment by dckesler** [comment]
Yes it should. The yarn.lock no longer has the vulnerable tar version.

**Pull request merged**